.github/workflows/main.yml

```diff
@@ -15,6 +15,15 @@ jobs:
           python-version: "3.13"
       - run: uv run pre-commit run --all-files
 
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          # Fetch history and tags to enable correct versioning
+          fetch-depth: 0
+      - uses: hynek/build-and-inspect-python-package@v2
```

Comment:
it was a while since I set this up but I used tox in the past to ensure that tests ran with the actual wheels and not just with the source checkout. do you know what best way to run pytest against the built wheels rather than the source checkout? just create a minimal tox config and invoke pytest via tox instead of directly, I guess? :) would be cool to use the exact built wheels from this job but maybe it does not matter. (we do not need to tackle that in this PR, just something that came to mind)

Reply:
I'd just let tox rebuild instead of worrying about reusing these. I think the main benefit of this CI job is to have a place where you can easily check what will be part of the next release, which used to be a pain with setuptools + `MANIFEST.in`.

Reply:
I've applied my typical tox setup in #131. It works locally and I've also updated the CI jobs to use the tox setup.

Reply:
awesome, thanks a lot!